CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Splunk SDK for Python Changelog
 
+## Versiom 1.6.6
+
+### Bug fixes
+
+* Fix ssl verify to require certs when true
+* Make the explorer example compatible w/ Python 3
+* Add full support for unicode in SearchCommands
+* Add return code for invalid_args block
+* Fixed unicode xml decoding error
+* Added headers argument to get requests so that additional headers can be passed in
```

Comment:
are all of these categorized as bug fixes?

Reply:
let me break them out